package.json

```diff
@@ -16,6 +16,7 @@
     "emoji-mart": "3.0.1",
     "linkifyjs": "^2.1.9",
     "lodash-es": "^4.17.21",
+    "node-fetch": "^2.0.0",
     "preact": "^10.5.14",
```

Comment:
I've installed v2 because v3 is buggy (webpack pbls)
https://github.com/node-fetch/node-fetch/issues/1265